lib/background_worker/base.rb

```diff
@@ -53,6 +53,7 @@ def log(message, options = {})
     end
 
     class << self
+      attr_accessor :queue
```

Comment:
I would make this attr_reader we want consumers to use the queue_as method

Reply:
Fixed